Makefile

```diff
@@ -8,6 +8,18 @@ TAR := $(shell if which gnutar 1>/dev/null 2> /dev/null; then echo gnutar; else
 
 default: wannier post
 
+PREFIX ?= /usr
+
+install:
```

Comment:
Please add the `install` target to the `.PHONY` target at the end of the file - this ensures that `make install` works even if you create a file `install` in the current folder.

Reply:
Has been added to `.PHONY`.